dicebot/app/bot.py

```diff
@@ -2,12 +2,15 @@
 
 import asyncio
 import logging
+import os
 import sys
 
 from dicebot.app import engine
 from dicebot.core.client import Client
 from dicebot.logging.colored_log_formatter import ColoredLogFormatter
 
+from tldrwl.register import Register
```

Comment:
third party libs should go above dicebot libs, right?

Reply:
ah yeah, I spedrun and read it as discord and was making it alphabetical. I'll fix it